.github/workflows/integration_test.yaml

```diff
@@ -22,9 +22,10 @@ jobs:
               mbzirc_sim /bin/bash -c \
                 'Xvfb :1 -ac -noreset -core -screen 0 1280x1024x24 & \
                  export DISPLAY=:1.0 && \
+                 export DISPLAY_TEST=1 && \
```

Comment:
Does it make sense to keep `DISPLAY_TEST` after fixing the problems?

Reply:
ok yeah, let's enable display tests by default. Removed `DISPLAY_TEST` in 086e7b4 and a6cbd478b225bfd6105b01349c640bc75d9957c4